contrib/syslog-debun

```diff
@@ -31,25 +47,34 @@ pseao="ps -eao"
 dfk="df -k"
 dfh="df -h"
 duks="du -ks"
+grepq="grep -q"
+lddcmd="ldd"
 mount=mount
 varlimit=1000
 myplimit () { echo "Plimit query is not supported on this platform" >&3 ; }
 distpkgoffile () { echo "Package file search is not supported (yet) on this platform" >&3 ; }
 distpkgstatus () { echo "Package status query is not (yet) supported on this platform" >&3 ; }
 selftar="tar cf - ."
+gzipcmd="gzip -9"
 showdep="dpkg -S"
 pcapifparm=-i
 w=w
 vmstat=vmstat
 dmesg=dmesg
-tcpdump="tcpdump -p -s 1500 -w"
-getsyslogpids="pidof syslog-ng"
+tcpdumpcmd="tcpdump"
+tcpdumpopts="-p -s 1500 -w"
+getsyslogpids () { pidof syslog-ng ; }
 trace="strace -f"
-service_stop="/etc/init.d/syslog-ng stop"
-service_start="/etc/init.d/syslog-ng start"
-service_status="/etc/init.d/syslog-ng status"
-checkpid () { [ -d /proc/"$1" ]; }
-mywait () { jobs -p >${tmpdir}/sdn.jobs ; for i in `grep -v "\<$tailpid\>" ${tmpdir}/sdn.jobs` ; do wait $i ; done ; }
+initfile="/etc/init.d/syslog-ng"
+service_stop="${initfile} stop"
+service_start="${initfile} start"
+service_status="${initfile} status"
+#checkpid () { [ -d /proc/"$1" ]; }
```

Comment:
I can not speak in János's name, but I think I understand his feelings on this:
How he refactored it, seems a good idea. But haven't proved it's stability, portability and robustness yet.
So, if the old code is there in the comments, and you are in a situation, like they are in the support team, and you have to collect the information from the cu asap, then it can be handy, if you still have the old but proven code!
I also understand your feelings, that git remembers everything.
But, please consider these thoughts regarding this script:
This is the support team's swiss-army knife^Wcannon.